lib/commands/cloud-projects.ts

```diff
@@ -117,5 +111,73 @@ export class CloudExportProjectsCommand implements ICommand {
 		return true;
 	}
 }
-
 $injector.registerCommand("cloud|export", CloudExportProjectsCommand);
+
+export class ExportCommand extends CloudExportProjectsCommand implements ICommand {
+	constructor($project: Project.IProject,
+		protected $errors: IErrors,
+		protected $remoteProjectService: IRemoteProjectService,
+		protected $cloudProjectsService: ICloudProjectsService,
+		private $server: Server.IServer,
+		private $logger: ILogger,
+		private $httpClient: Server.IHttpClient,
+		private $fs: IFileSystem,
+		private $progressIndicator: IProgressIndicator,
+		private $options: IOptions) {
+		super($errors, $remoteProjectService, $cloudProjectsService, $project);
+	}
+
+	public allowedParameters: ICommandParameter[] = [];
+	protected _shouldCheckProject = false;
+
+	public async execute(args: string[]): Promise<void> {
+		const solutionProjectInfo = await this.$cloudProjectsService.getSolutionProjectInfo({
+			projectName: args[1],
+			solutionName: args[0],
+			forceChooseProject: true
+		});
+
+		const properties = {
+			Framework: solutionProjectInfo.framework,
+			AcceptResults: "Url;LocalPath"
+		};
+
+		this.$logger.info("Exporting project...");
+		// Fail early if the download path exists
+		let downloadPath = resolve(this.$options.path || ".");
+		const downloadPathExists = this.$fs.exists(downloadPath);
+		let downloadPathIsDirectory = true;
+		if (downloadPathExists) {
+			downloadPathIsDirectory = this.$fs.getFsStats(downloadPath).isDirectory();
+			if (!downloadPathIsDirectory) {
+				this.$errors.failWithoutHelp(`Cannot download result package in ${downloadPath} as it already exists.`);
+			}
+		}
+
+		const exportPromise = this.$server.appsBuild.exportProject(solutionProjectInfo.id, solutionProjectInfo.projectName, { Properties: properties, Targets: [] });
+		const response = await this.$progressIndicator.showProgressIndicator<Server.BuildResultData>(exportPromise, 2000);
+		if (response.Errors.length) {
+			this.$errors.failWithoutHelp(`Export errors:${EOL}${response.Errors.map(e => e.Message).join(EOL)}`);
+		} else {
+			const buildItem = response.ResultsByTarget["Build"].Items[0];
+			const downloadUrl = buildItem.FullPath;
+			if (!downloadPathExists || downloadPathIsDirectory) {
+				if (extname(downloadPath)) {
+					this.$fs.ensureDirectoryExists(dirname(downloadPath));
+				} else {
+					this.$fs.ensureDirectoryExists(downloadPath);
+					downloadPath = join(downloadPath, buildItem.Filename);
+				}
+			}
+
+			this.$logger.info(`Downloading ${buildItem.Filename} to ${downloadPath}...`);
+			const targetFile = this.$fs.createWriteStream(downloadPath);
+			await this.$httpClient.httpRequest({
```

Comment:
In case this request fails, it will still create a targetFile with size of 0KB. I suggest to try - catch this call and delete the targetFile in case operation fails.